keyring/backend.py

```diff
@@ -127,19 +127,12 @@ def decrypt(self, value):
         return value
 
 
-def _load_backend(name):
-    "Load a backend by name"
-    package = backends.__package__ or backends.__name__
-    mod = importlib.import_module('.' + name, package)
-    # invoke __name__ on each module to ensure it's loaded in demand-import
```

Comment:
This important behavior is flagged with a comment to prevent it from being inadvertently optimized away, which this change does. So unfortunately, this change will break keyring in environments where demand-import is used (Mercurial in particular).